### PR TITLE
Update the page on "Google Pub Sub Connector Example"

### PR DESCRIPTION
**Pull request by poorna2152**
## Purpose
$subject

### CODE REVIEW

File: en/docs/reference/connectors/google-pubsub-connector/googlepubsub-connector-example.md [hunk 1]
```diff
@@ -32,159 +32,151 @@ The following diagram shows the overall solution. The user creates a topic, stor
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-## Configure the connector in WSO2 Integration Studio
+## Setup the Integration Project
```
Comment (gabilang) [inline en/docs/reference/connectors/google-pubsub-connector/googlepubsub-connector-example.md:35]:
```suggestion
## Setting Up the Integration Project
```

File: en/docs/reference/connectors/google-pubsub-connector/googlepubsub-connector-example.md [hunk 2]
```diff
@@ -12,179 +12,171 @@ Given below is a sample scenario that demonstrates how to work with the WSO2 Goo
...
+         <img src="{{base_path}}/assets/img/integrate/connectors/gpubsub/pubsub-api-createtopicsubscription-operation.png" title="Add values to the createTopicSubscription operation" width="800" alt="Add values to the createTopicSubscription operation"/>
+
+    2. Add the property mediator to capture the `subscriptionName` values. This contains the name of the subscription.
+
+        - **name** : `subscriptionName`
+        - **expression** : `json-eval($.subscriptionName)`
+
+        <img src="{{base_path}}/assets/img/integrate/connectors/gpubsub/pubsub-api-property-mediator-property2-value2.png" title="Add values to capture subscriptionName" width="800" alt="Add values to capture subscriptionName"/>  
+
+    3. Add the property mediator to store the name of the created Topic value from the response of the createTopic operation.
```
Comment (chanikag) [inline en/docs/reference/connectors/google-pubsub-connector/googlepubsub-connector-example.md:109]:
```suggestion
    3. Add the property mediator to store the name of the created Topic value from the response of the `createTopic` operation. 
```

File: en/docs/reference/connectors/google-pubsub-connector/googlepubsub-connector-example.md [hunk 3]
```diff
@@ -12,179 +12,171 @@ Given below is a sample scenario that demonstrates how to work with the WSO2 Goo
...
+        <img src="{{base_path}}/assets/img/integrate/connectors/gpubsub/pubsub-api-property-mediator-property2-value2.png" title="Add values to capture subscriptionName" width="800" alt="Add values to capture subscriptionName"/>  
+
+    3. Add the property mediator to store the name of the created Topic value from the response of the createTopic operation. 
+
+        - **name** : `nameforsubscription`
+        - **expression** : `json-eval($.name)`
+
+        <img src="{{base_path}}/assets/img/integrate/connectors/gpubsub/pubsub-api-property-mediator-nameforsubscription.png" title="Add values to capture nameforsubscription" width="800" alt="Add values to capture nameforsubscription"/>
+
+    4. Add the property mediator to capture the Topic name from the response using the splitting separators in the results.
```
Comment (chanikag) [inline en/docs/reference/connectors/google-pubsub-connector/googlepubsub-connector-example.md:116]:
```suggestion
    4. Add the property mediator to capture the topic name from the response using the splitting separators in the results.  
```

File: en/docs/reference/connectors/google-pubsub-connector/googlepubsub-connector-example.md [hunk 4]
```diff
@@ -275,24 +264,11 @@ You can download the ZIP file and extract the contents to get the project code.
```
Comment (chanikag) [inline en/docs/reference/connectors/google-pubsub-connector/googlepubsub-connector-example.md:264]: Is this zip file updated?

**Review by chanikag — APPROVED** [review]

**Pull request merged**